atlp-rwanda/e-comm-team-brogrammers-bn
# #184581360 adding notifications

#### What does this PR do?

This PR id for adding notifications

#### Description of Task to be completed?

this task is composed of making sure that the user gets correct notification according to actions made on products

#### How should this be manually tested?

-first of all clone the repo and  run `npm run migrate:all`
- then login and put token in bearer
- you can check your notifications on `/notifications` with GET method
-  you can delete your notification `/notifications/notificationId` or even clear using `/notifications/clear` with patch  

#### Any background context you want to provide?

n/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184581360

#### Screenshots (if appropriate)
<img width="777" alt="Screenshot 2023-03-30 at 09 36 43" src="https://user-images.githubusercontent.com/99663528/228773818-911e0218-236e-4749-aaa5-c6c07c135c66.png">
<img width="1391" alt="Screenshot 2023-03-29 at 17 55 31" src="https://user-images.githubusercontent.com/99663528/228773831-6d44d123-3155-4968-afab-642835b05d58.png">

#### Questions:
n/a
